### PR TITLE
Fix: Scrolling

### PR DESCRIPTION
**Pull request by atstojanov**
### Describe the background of your pull request

What does your pull request do? Does it solve a bug (which one?), add a feature?

### Additional context

Add any other context

### Governance 

- [ ] Documentation is added
- [ ] Test cases are added or updated

By submitting this pull request I confirm that:
- I've read the contributing document https://github.com/alkem-io/.github/blob/master/CONTRIBUTING.md.
- I've read and understand the Code of Conduct https://github.com/alkem-io/.github/blob/master/CODE_OF_CONDUCT.md.
- I understand that any contributions or suggestions I made may make it into the actual code. I've read the License 
     https://github.com/alkem-io/Coordination/blob/master/LICENSE.
- I understand that submitting the pull request requires agreeing to and signing the contributor license agreement.

### CODE REVIEW

File: src/components/core/ScrollButton/ScrollButton.tsx [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+  };
+
+  useEffect(() => {
+    const toggleVisible = () => {
+      const scrolled = document.documentElement.scrollTop;
+      if (scrolled > 30) {
+        setVisible(true);
+      } else if (scrolled <= 30) {
+        setVisible(false);
+      }
```
Comment (hero101) [inline src/components/core/ScrollButton/ScrollButton.tsx:26]: why `30` exactly ??
also can be simplified to `setVisible(scrolled > 30)`

File: src/pages/About.tsx [hunk 2]
```diff
@@ -52,6 +53,10 @@ const AboutPage = () => {
   const styles = useAboutStyles();
   const { data } = useServerMetadataQuery();
   const { t } = useTranslation();
+
+  const currentPaths = useMemo(() => [], []);
```
Comment (hero101) [inline src/pages/About.tsx:57]: why is `useMemo` used here when it can be a constant?

File: src/root.tsx [hunk 3]
```diff
@@ -68,6 +69,7 @@ const Root: FC = () => {
                     <NavigationProvider>
                       <UserProvider>
                         <App>
+                          <ScrollToTop />
```
Comment (hero101) [inline src/root.tsx:72]: Is this really the proper solution? Seems kind of strange.

**Review by hero101 — CHANGES_REQUESTED** [review]

File: src/components/core/ScrollButton/ScrollButton.tsx (hunk 1 above)
Comment (atstojanov) [reply src/components/core/ScrollButton/ScrollButton.tsx:26]: No reason at all.

File: src/pages/About.tsx (hunk 2 above)
Comment (atstojanov) [reply src/pages/About.tsx:57]: have to be memoised otherwise the breadcrumb will trigger infinity rerendering.

Comment (hero101) [reply src/pages/About.tsx:57]: Even on constants?

Comment (atstojanov) [reply src/pages/About.tsx:57]: It is possible to be constant if is defined outside the scope of the component.

**Review by hero101 — APPROVED** [review]

**Pull request merged**